### DISCUSSION TITLE
tauri-apps/api, how to decode the content of ```WebviewWindow```

### DISCUSSION THREAD

**Discussion by yk0n9** [tauri-apps/tauri#7590] [Q&A]
my code:
```javascript
let webview = new WebviewWindow("document", {
                      url: `data:text/html,<html>你好</html>`,
                      title: intl.get("documents"),
});
```
i want to write some custom html content in string instead of fetching from real url
In other words, I want to construct a ```html``` from the js string and display it in ```WebviewWindow```

**Comment by FabianLars** [comment]
Did you add the `window-data-url` feature flag to the tauri dependency in cargo.toml? If you did, it _should_ work like in your code snippet.

**Reply by yk0n9** [reply]
thank you, i will try it
